### ISSUE TITLE
PHP 8.1 compatibility

### ISSUE THREAD

**Issue by dd32** [Automattic/gutenberg-everywhere#141]
Activating the plugin fails under PHP 8.1, Switching back to PHP 7.4 resolves it and allows activation of the plugin.

I suspect this might be either load-order of the files, or potentially something about the Class name matching the Namespace (But I would expect widespread breakage in other plugins/apps if that were the case)

```
PHP Fatal error:  Uncaught Error: Class "Automattic\Blocks_Everywhere\Handler\Handler" not found in wp-content/plugins/blocks-everywhere/classes/handlers/class-bbpress.php:6
Stack trace:
#0 wp-content/plugins/blocks-everywhere/classes/class-handler.php(7): require_once()
#1 wp-content/plugins/blocks-everywhere/blocks-everywhere.php(14): require_once()
#2 wp-admin/includes/plugin.php(2314): include_once()
#3 wp-admin/includes/plugin.php(661): plugin_sandbox_scrape('blocks-everywhe...')
#4 wp-admin/plugins.php(58): activate_plugin('blocks-everywhe...', 'https://sv.word...', false)
#5 {main}
  thrown in wp-content/plugins/blocks-everywhere/classes/handlers/class-bbpress.php on line 6
```

**Comment by johngodley** [comment]
I've been using PHP 8.1 to develop the plugin so this is very strange! Maybe there are some config settings that affect compatibility.

I've released 1.14.3 to change how the files are included. I think this should fix the problem, but can't confirm myself.

**Comment by dd32** [comment]
The strange thing was that this was only upon plugin activation, running the plugin was fine, it just couldn't be activated. 

I can't duplicate it now with the new version, but reverting to an older version I can't duplicate my original fatal either... so.. I'm not sure what to make of that.. I could duplicate it pretty consistently yesterday..

Closing this ticket, either it's a problem on my end, or the above fixed it.
